src/components/DataTable.tsx

```diff
@@ -220,7 +338,7 @@ export function DataTable<T extends object>(props: Props<T>) {
     ],
     () => request('GET', apiEndpoint.href),
     {
-      staleTime: props.staleTime || 5000,
+      staleTime: props.staleTime ?? Infinity,
```

Comment:
What's the reason behind this change?

Reply:
the 5second staletime was not providing a good experience, and with the change to remembering the user preferences for filtering, it was also causing a double query to be performed. It was decided than with the new invalidation being more aggressive, we could push the staletime out to Infinity.

Reply:
@beganovich @turbo124 Yep! So, in fact, a `staleTime` of 5 seconds does not make sense unless our invalidation works perfectly fine. We actually needed invalidation even when the `staleTime` was set to 5 seconds in order to enable tables to be updated immediately after calling certain actions directly from the table.

We discussed this briefly on Slack. So, basically, this will significantly reduce the number of API calls.